docs/cluster_spec.md

```diff
@@ -0,0 +1,444 @@
+| peerVerificationRules | PeerVerificationRules provides the rules for what client certificates the process should accept. | string | false |
+| env | Env provides environment variables.  **Deprecated: Use the PodTemplate field instead.** | [][corev1.EnvVar](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#envvar-v1-core) | false |
+| volumeMounts | VolumeMounts provides volume mounts.  **Deprecated: Use the PodTemplate field instead.** | [][corev1.VolumeMount](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#volumemount-v1-core) | false |
+| imageName | ImageName provides the name of the image to use for the container, without the version tag.  **Deprecated: Use the PodTemplate field instead.** | string | false |
+| securityContext | SecurityContext provides the container's security context.  **Deprecated: Use the PodTemplate field instead.** | *[corev1.SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#securitycontext-v1-core) | false |
+
+[Back to TOC](#table-of-contents)
+
+## DataCenter
+
+DataCenter represents a data center in the region configuration
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| id | The ID of the data center. This must match the dcid locality field. | string | false |
+| priority | The priority of this data center when we have to choose a location. Higher priorities are preferred over lower priorities. | int | false |
+| satellite | Satellite indicates whether the data center is serving as a satellite for the region. A value of 1 indicates that it is a satellite, and a value of 0 indicates that it is not a satellite. | int | false |
+
+[Back to TOC](#table-of-contents)
+
+## DatabaseConfiguration
+
+DatabaseConfiguration represents the configuration of the database
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| redundancy_mode | RedundancyMode defines the core replication factor for the database. | string | false |
+| storage_engine | StorageEngine defines the storage engine the database uses. | string | false |
+| usable_regions | UsableRegions defines how many regions the database should store data in. | int | false |
+| regions | Regions defines the regions that the database can replicate in. | [][Region](#region) | false |
+| RoleCounts | RoleCounts defines how many processes the database should recruit for each role. | [RoleCounts](#rolecounts) | true |
+
+[Back to TOC](#table-of-contents)
+
+## FdbVersion
+
+FdbVersion represents a version of FoundationDB.  This provides convenience methods for checking features available in different versions.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| Major | Major is the major version | int | false |
+| Minor | Minor is the major version | int | false |
+| Patch | Patch is the major version | int | false |
```

Comment:
Looks like the documentation has some copy/paste errors. It might be worth fixing it in the external file

Reply:
Otherwise everything looks good and ready to merge